From 5d3d72bf3fd6d72a50001b76893e4130a022e5fd Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:08:27 -0400
Subject: [tools] perf: Add warning when disabling perl scripting support due to missing devel files

Message-id: <1410181752-25631-263-git-send-email-jolsa@redhat.com>
Patchwork-id: 91841
O-Subject: [PATCH RHEL7.1 BZ1134356 262/307] perf tools: Add warning when disabling perl scripting support due to missing devel files
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 6a2f2543a1f3aa0e7766e27c30b93d164771e892
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Mon May 26 16:02:30 2014 -0300

description
===========
We were just showing "libperl: OFF", unlike other features where we
present the user with a message helping have a feature built in.

Fix it by adding the following message:

config/Makefile:450: Missing perl devel files. Disabling perl scripting support, consider installing perl-ExtUtils-Embed
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/config/Makefile b/tools/perf/config/Makefile
index 729bbdf..319426f 100644
--- a/tools/perf/config/Makefile
+++ b/tools/perf/config/Makefile
@@ -447,6 +447,7 @@ else
   ifneq ($(feature-libperl), 1)
     CFLAGS += -DNO_LIBPERL
     NO_LIBPERL := 1
+    msg := $(warning Missing perl devel files. Disabling perl scripting support, consider installing perl-ExtUtils-Embed);
   else
     LDFLAGS += $(PERL_EMBED_LDFLAGS)
     EXTLIBS += $(PERL_EMBED_LIBADD)
-- 
1.7.1